data-mongodb/src/main/java/io/micronaut/data/mongodb/serde/DataDecoderContext.java

```diff
@@ -213,7 +218,7 @@ public void popManagedRef() {
 
     @Override
     public boolean hasView(Class<?>... views) {
-        return true;
+        return this.mongoDataConfiguration.isIgnoreJsonViews();
```

Comment:
Please delegate this one to the parent as well.

Reply:
> Please delegate this one to the parent as well.

Done